depthai_helpers/arg_manager.py

```diff
@@ -110,7 +110,7 @@ def parseArgs():
                         help="Mono cam fps: max 60.0 for H:720 or H:800, max 120.0 for H:400. Default: %(default)s")
     parser.add_argument('-cb', '--callback', type=Path, default=projectRoot / 'callbacks.py', help="Path to callbacks file to be used. Default: %(default)s")
     parser.add_argument("--openvinoVersion", type=str, choices=openvinoVersions, help="Specify which OpenVINO version to use in the pipeline")
-    parser.add_argument("--app", type=str, help="Specify which app to run instead of the demo")
+    parser.add_argument("-app","--app", type=str, help="Specify which app to run instead of the demo")
```

Comment:
May be helpful to mention also the list of available apps (`uvc` for now, later we'll add also `tof`).